encord/user_client.py

```diff
@@ -847,13 +847,17 @@ def get_storage_item(self, item_uuid: Union[UUID, str], sign_url: bool = False)
             item_uuid = UUID(item_uuid)
         return StorageItem._get_item(self._api_client, item_uuid, sign_url)
 
-    def get_storage_items(self, item_uuids: List[Union[UUID, str]], sign_url: bool = False) -> List[StorageItem]:
+    def get_storage_items(
+        self,
+        item_uuids: Sequence[Union[UUID, str]],
+        sign_url: bool = False,
+    ) -> List[StorageItem]:
         """
         Get storage items by their UUIDs, in bulk. Useful for retrieving multiple items at once, e.g. when getting
         items pointed to by :attr:`encord.orm.dataset.DataRow.backing_item_uuid` for all data rows of a dataset.
 
         Args:
-            item_uuids: list of UUIDs of items to retrieve.
+            item_uuids: list of UUIDs of items to retrieve. Can be a list of strings or a list of UUID objects.
```

Comment:
```suggestion
            item_uuids: list of UUIDs of items to retrieve. 
            Either UUIDs or string representations of UUIDs are accepted.
```
Slightly prefer this to follow others + make its clear that we can handle a mixture. As written, not clear that it handles a mixture, looks disjoint.